roles/prune_catalog/tasks/main.yml

```diff
@@ -21,77 +22,82 @@
       --tls-verify=false docker://{{ pc_source_catalog }}
     fi
   register: skopeo_result
+  changed_when: skopeo_result.rc != 0
   retries: 3
   delay: 10
   until: skopeo_result.rc == 0
   failed_when:
     - skopeo_result.rc != 0
     - not pc_ignore_pull_errors
 
-- block:
-    - name: Create temporary directory
-      tempfile:
-        state: directory
-        prefix: pc_tmp_dir.
-      register: pc_tmp_dir
+- name: "Prune catalog"
+  when: skopeo_result.rc == 0
+  block:
+    # - name: Create temporary directory
```

Comment:
Is this code (and the rest of commented code) commented for a specific reason? Can it be deleted?